adding_metadata_sql.md

```diff
@@ -0,0 +1,193 @@
+## SQL
+
+SQL can also be used for bulk uploads. However, SQL should primarily be used in the following cases:
+
+1. correcting systemic data errors involving large numbers of rows
+2. assigning non-point geometries to sites [this might be possible in YABA, I don't know]
+3. associating experiments with sites and treatments
+4. associating sites with cultivars
+
+### Metadata Entry Workflow for BETYdb
+
+The following steps have been implemented in the BETYdb-YABA API and python client, but are described below for reference
+
+Here we show SQL code used to add metadata required each season by the TERRA REF database, including experiments, sites, treatments, cultivars, and citations that is required prior to uploading the trait or yield data. Most PEcAn users _will not_ need to add experiments and cultivars, or associate these records with sites.
+
+Also note that in the TERRA REF and other agronomic applications, each record in the 'sites' table will correspond to an experimental plot.
+
+### Step 1: Add new experiments
+
+```sql
+insert into experiments (name, start_date, end_date, user_id) 
+values ('MAC Season 6: Sorghum BAP', '2018-04-06', '2018-08-01', 'some text', 'some text', 6000000004);
+```
+
+### Step 2: Add new sites
+
+_must provide shapefile for a site to get geometry_
```

Comment:
```suggestion
To add sites (or many plots at a single site) using the BETYdb-YABA API, you must provide shapefile for a site to have an associated geometry. 

Otherwise, you can add sites with points or polygons to the database as follows.
```